src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -452,7 +451,7 @@ def aks_create(
     # addons
     enable_addons=None,
     workspace_resource_id=None,
-    enable_msi_auth_for_monitoring=False,
+    enable_msi_auth_for_monitoring=True,
```

Comment:
May I ask if this change of default value will cause a breaking change to customers?

Reply:
@zhoxing-ms thank you for checking this. This is not counted as breaking change since it won't affect existing customers and only applicable for new customers.

Reply:
@zhoxing-ms, As @wanlonghenry  mentioned, we are making this as default  for new customers and there is no impact to the existing customers. Do you want to add this in the release notes of the CLI version which will have this change? if yes, can you please share the release notes which we would need to add.